README.md

```diff
@@ -1,39 +1,68 @@
 
-Not all functionality has been implemented yet and there may be significant changes to both this library and the protocol.
+* Define workflows in code. No JSON schemas or designers are needed.
+* Call other functions synchronously and asynchronously. Output from called functions can be saved to local variables.
+* Automatically checkpoint progress whenever the function schedules async work. Local state is never lost if the process recycles or the VM reboots.
+
+You can find more information at the following links:
+
+* [Azure Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview)
+* [Azure Functions JavaScript developers guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node)
+* [Durable Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview)
+
+A durable function, or _orchestration_, is a solution made up of different types of Azure Functions:
+
+* **Activity:** the functions and tasks being orchestrated by your workflow.
+* **Orchestrator:** a function that describes the way and order actions are executed in code.
+* **Client:** the entry point for creating an instance of a durable orchestration.
+
+Durable Functions' function types and features are documented in-depth [here.](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview)
 
 ## Getting Started
 
-1. Install Durable Functions
+You can follow the [Visual Studio Code quickstart](https://docs.microsoft.com/en-us/azure/azure-functions/durable/quickstart-js-vscode) to get started with a function chaining example, or follow the general checklist below:
 
-Run this command from the root folder of your functions app:
-```
+1. Install prerequisites:
+    - [Azure Functions Core Tools version 2.x](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#install-the-azure-functions-core-tools)
+    - [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator) (Windows) or an actual Azure storage account (Mac or Linux)
+    - Node.js 8.6.0 or later
+
+2. [Create an Azure Functions app.](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-vs-code) Visual Studio Code is recommended.
+
+3. Install the Durable Functions extension
+
+Run this command from the root folder of your Azure Functions app:
+```bash
 func extensions install -p Microsoft.Azure.WebJobs.Extensions.DurableTask -v 1.7.0
 ```
 
-2. Install the package
+4. Install the `durable-functions` npm package
```

Comment:
Will this still work if installed globally, or does it need to be installed locally and deployed with your function?